cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1712,19 +1733,6 @@ void SubstraitToVeloxPlanConverter::separateFilters(
   for (const auto& scalarFunction : scalarFunctions) {
     auto filterNameSpec = SubstraitParser::findFunctionSpec(functionMap_, scalarFunction.function_reference());
     auto filterName = SubstraitParser::getNameBeforeDelimiter(filterNameSpec);
-    // Add all decimal filters to remaining functions because their pushdown are not supported.
-    if (format == dwio::common::FileFormat::ORC && scalarFunction.arguments().size() > 0) {
```

Comment:
Looks like previous logic only excludes decimal type for ORC format, but now we are excluding HUGEINT for all file formats. Could you confirm if it is expected?

Reply:
The ORC format already supports short-decimal predicate pushdown. @jiangjiangtian Can you help test this?

Reply:
The main idea of these three types (`Timestamp/VarBinary/HugeInt`) in this PR is that if we don't put them in `remainingFilter`, then when we call `mapToFilters`, a exception `Subfield filters creation not supported for input type '{}' in mapToFilters`  will throw, which will cause scan fallback. 

The reason why `mapToFilters` don't support these three types is because there is no `MultiRangeType` or related type traits defined, which has nothing to do with the fileformat. cc @rui-mo @jiangjiangtian @kecookier

Reply:
That means actually we don't support the pushdown of long decimal in Parquet and ORC. Is that correct?

Reply:
> That means actually we don't support the pushdown of long decimal in Parquet and ORC. Is that correct?

@rui-mo Currently yes. But to support a `HugeintMultiRange` in velox side is not a big issue, I can work on it later.